vrc-get-gui/src/templates/alcom_template.rs

```diff
@@ -106,19 +106,32 @@ pub fn parse_alcom_template(json: &[u8]) -> serde_json::Result<AlcomTemplate> {
     let template = serde_json::from_slice::<AlcomTemplateContent>(json)?;
 
     // few validations
-    if let Some(id) = &template.id {
-        if id.0.starts_with("com.anatawa12.vrc-get") {
+    if let Some(id_wrapper) = &template.id { // id_wrapper is &TemplateId
+        // Reserve `com.anatawa12.vrc-get.*` except for known namespaces which are allowed for user-created
+        // templates (".user") and imported VCC templates (".vcc").
+        if id_wrapper.0.starts_with("com.anatawa12.vrc-get")
+            && !(id_wrapper.0.starts_with("com.anatawa12.vrc-get.user.")
+                || id_wrapper.0.starts_with("com.anatawa12.vrc-get.vcc."))
+        {
             return Err(serde_json::Error::invalid_value(
-                Unexpected::Str(&id.0),
-                &"a valid alcom template id (reserved id)",
+                Unexpected::Str(&id_wrapper.0),
+                &"a valid alcom template id (reserved id, or disallowed com.anatawa12.vrc-get.* namespace)",
```

Comment:
"disallowed com.anatawa12.vrc-get.* namespace" is part of reserved id so we don't have to add message